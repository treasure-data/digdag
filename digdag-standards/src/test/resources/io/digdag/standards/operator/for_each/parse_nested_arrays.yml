_command:
  a: '[1,2]'
  b: '[3,4]'
_do:
  echo>: value is ${a} and ${b}
