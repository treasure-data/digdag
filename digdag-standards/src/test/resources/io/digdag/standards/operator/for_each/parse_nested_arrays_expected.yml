+for-0=a=0=1&1=b=0=3:
  echo>: value is ${a} and ${b}
  _export: {a: 1, b: 3}
+for-0=a=0=1&1=b=1=4:
  echo>: value is ${a} and ${b}
  _export: {a: 1, b: 4}
+for-0=a=1=2&1=b=0=3:
  echo>: value is ${a} and ${b}
  _export: {a: 2, b: 3}
+for-0=a=1=2&1=b=1=4:
  echo>: value is ${a} and ${b}
  _export: {a: 2, b: 4}
